### ISSUE TITLE
Check logdir path

### ISSUE THREAD

**Issue by gyadam** [tensorflow/tensorboard#3397]
### Version:
TensorBoard 2.1.0

### Issue:
TensorBoard doesn't check if ```logdir``` exists or not.
Not a huge issue, but a bit weird that there is no error message if I provide a non-existing directory. If one mistakenly provides a faulty path, they will probably only notice it when opening the link in the browser, after which they have to close the tab, go back, kill the process and restart it. It would be nice to have a warning at least.

### Suggestion
Start Tensorboard as usual, but warn the user if ```logdir``` does not exist (yet).

**Comment by bileschi** [comment]
This is a good idea.  I've also started tensorboard pointing at a directory that exists, but made a typo in the invocation

**Comment by bileschi** [comment]
Some infrastructure will start a tensorboard pointing a location that doesn't exist yet, and then (eventually) start to fill that location with summaries.  I wonder if a warning would be a problem for those users.

Routing to @GalOshri for guidance

**Comment by wchargin** [comment]
Yep, that’s why we’ve historically not done this, and in fact considered
it a blocker for designs that would rely on this, such as better
diagnostics for accidental misuse of `--logdir_spec`.

For instance, in Colab/Jupyter, one runs `%tensorboard --logdir ...`
prior to starting the model training, because while the model is
training you can’t execute any other Colab cells, but if you’ve already
started TensorBoard then it will continue to live-update.

**Comment by gyadam** [comment]
Yeah, that's what I was thinking. Should I close the issue?

**Comment by bileschi** [comment]
There could be some avenue for improving the usability.  Perhaps a flag or config to allow users to specify the desired behavior.  What designs wouldn't interrupt current behavior but would make lives easier?
